.tekton/pull-request.yaml

```
@@ -163,6 +163,82 @@ spec:
         workspaces:
           - name: source
             workspace: workspace
+      - name: get-tasks-to-evaluate
+        runAfter:
+          - fetch-repository        
+        taskSpec:
+          results:
+          - description: Directory containing the build tasks
+            name: build_tasks-ec
+          - description: Directory containing all tasks
+            name: all_tasks-ec
+          steps:
+          - name: gather-tasks
+            image: quay.io/jstuart/fedora-kustomize:latest
```

Just to check the, this won't work here?

```suggestion
            image: registry.redhat.io/openshift4/ose-tools-rhel8:v4.12
```